swift-generator-cli/src/main/java/com/facebook/swift/generator/SwiftGeneratorCommandLineConfig.java

```diff
@@ -86,4 +86,10 @@
             description = "Include paths are relative to the file in which they are contained"
     )
     public boolean includePathsRelativeToFile = false;
+
+    @Parameter(
+            names = "-package_prefix",
```

Comment:
Since this is `java` specific, perhaps the name of the parameter should be `java_package_prefix`.